sgx-tools/gen-qe-target-info.go

```diff
@@ -27,12 +37,35 @@ For example, save the target information file about Quoting Enclave retrieved fr
 			logrus.SetLevel(logrus.DebugLevel)
 		}
 
-		ti, err := intelsgx.GetQeTargetInfo()
+		var ti []byte
+		var err error
+
+		serviceType := context.String("serviceType")
```

Comment:
It looks like below codes should be encapsulated into GetQeTargetInfo() API provided by intelsgx package. This API should accept a parameter called "quote-type".

Even you don't want to do this immediately, you should encapsulate them into a separate function to show you have already considered this final design but currently not. Showing your positive motivation and intention is very important through rfc codes or even just comments.